### ISSUE TITLE
Updating new headers

### ISSUE THREAD

**Issue by katz95** [spotify/backstage#1232]
## 🗒 General
Hi! Would love some help updating the colors of our headers. We've made some adjustments to ensure accessibility. We've also added some additional headers for different components. If you all could update our [header section in Storybook](http://storybook.backstage.io/?path=/story/header--website) too, that would be rad!

![Colors](https://user-images.githubusercontent.com/61153904/84297167-63000a80-ab4d-11ea-86b0-e4a40de333f1.png)

## 💻 Usage
Headers are fixed to the top of Backstage - they depict what you're looking at. There's additional functionality that can be added to headers, such as breadcrumbs, links, statuses, etc. For now, we've included the generic version in the Figma working file.

## 📐 Specs
We're going to try something new with specs this time, and we'll be leveraging our Figma community file! 
<img width="1489" alt="Screenshot 2020-06-10 at 7 12 52 PM" src="https://user-images.githubusercontent.com/61153904/84297785-6942b680-ab4e-11ea-8f4f-0be74891a800.png">

Steps:
1) Duplicate the file [here](https://www.figma.com/@backstage). Take a look through it and check out the README files by clicking on the 'README' section on the lefthand side of the app.
2) Click into the 'Header' page.
<img width="1674" alt="Screenshot 2020-06-10 at 7 11 13 PM" src="https://user-images.githubusercontent.com/61153904/84297879-87a8b200-ab4e-11ea-8872-519d8cda4cbb.png">
3) You should be able to click on the different parts of the components to see their specs.
<img width="1674" alt="Screenshot 2020-06-10 at 7 10 12 PM" src="https://user-images.githubusercontent.com/61153904/84297899-90998380-ab4e-11ea-9776-38e5d1d8db59.png">

## 🔮 Future
We'll keep ya updated on additional features/functionality for headers! As we noted above, there's additional functionality that can be added to headers, such as breadcrumbs, links, statuses, etc. More to come!

**Comment by benjaminr** [comment]
Happy to help out on this one. Will look at putting together a PR.

**Comment by Rugvip** [comment]
@katz95 it seems like this design increases the number of different colors involved in each page theme, is that correct? Currently we have 2 colors defined, and this seems to increase it to 4.

I think the best case would be if we can stick to 2 colors, and then define the gradients in terms of mixes of those 2 colors, would that be possible? If that's not enough we could also bring in a black or white layer as background or foreground with different opacities.

**Comment by katz95** [comment]
hmm, @Rugvip, the gradients are one color (just at different opacities), and the backgrounds are either one color or two colors combined into a gradient. if necessary, i can make the background into just one solid color!

**Comment by stefanalund** [comment]
@katz95 what is the status of this? Has it been fixed?

**Comment by katz95** [comment]
can we use 2-color gradients in the background like this and have one color gradients for the waves? let me know! @Rugvip @benjaminr 
![Colors](https://user-images.githubusercontent.com/61153904/89778841-79efb880-db0e-11ea-9996-c491d4bf0876.png)
![Colors](https://user-images.githubusercontent.com/61153904/89778756-54fb4580-db0e-11ea-8f35-c3a463370ad7.png)

**Comment by kingasieminiak** [comment]
H! @Rugvip @katz95  Did you think of enabling headers' color modification through the theme (for the app created with CLI)? Once someone adjusts the theme color palette to match their branding, the colors of headers (especially the home one) might not fit perfectly. Of course, there is always a way to not use them, but then there might be an inconsistency between the customized app and open-source plugin that will be used by this app.

And since colors passed to the header are quite similar, you might also consider providing only base color and generate other ones using lighten and darken helpers 🙃 .

**Comment by Rugvip** [comment]
@kingasieminiak yeah that's what I've been aiming for here 😅

**Comment by katz95** [comment]
@kingasieminiak @Rugvip ahhh gotcha! that's a good point. hmm so from a design perspective, do you just need me to provide base colors then? because in that case, you can just grab the first HEX value from each of the base gradients that i've listed in the image above!

**Comment by kingasieminiak** [comment]
@katz95  Yes, that would do. But please, notice that some of the second background colors (e.g. website and tool gradient) are not exactly the dark version of a first one, but slightly different color. If we pick all colors based on the first one, the result might be not exactly the same that the one specified in Figma.

I can take that ticket if there is no one else working on that, so you will be able to see if the result is satisfying for you 😃

**Comment by nikek** [comment]
I've restructured the SVGs in Figma a bit so we only need to change the background colors. The actual wave shapes are now white to transparent gradient masks. @katz95 will move them to the public Figma when she has time. I will can implement them meanwhile.

**Comment by nikek** [comment]
A PR for the interested: https://github.com/spotify/backstage/pull/2015
